pkg/unikontainers/unikernels/rumprun.go

```diff
@@ -75,42 +75,51 @@ func (r Rumprun) CommandString() (string, error) {
 	return jsonStr, nil
 }
 
-func newRumprun(data UnikernelParams) (Rumprun, error) {
-	tempBlk := RumprunBlk{
-		Source:     "etfs",
-		Path:       "/dev/ld0a",
-		FsType:     "blk",
-		Mountpoint: "/data",
+func (r *Rumprun) SupportsBlock() bool {
+	return true
+}
+
+func (r *Rumprun) SupportsFS(fsType string) bool {
+	switch fsType {
+	case "ext2":
+		return true
+	default:
+		return false
 	}
+}
+
+func (r *Rumprun) Init(data UnikernelParams) error {
```

Comment:
Likewise, we could remove the empty line.

Reply:
Sure